site/blocks/front-matter.scss

```diff
@@ -0,0 +1,103 @@
+.front-matter {
+    margin-top: 48px;
+    display: grid !important;
+    grid-template-columns: repeat(12, 1fr);
+    column-gap: 24px;
+
+    .wp-block-column:nth-child(1) {
+        grid-column: 1 / 13;
+
+        @media (min-width: 960px) {
+            grid-column: 1 / 9;
+        }
+        @media (min-width: 1280px) {
+            grid-column: 1 / 6;
+        }
+    }
+
+    .wp-block-column:nth-child(2) {
+        grid-column: 1 / 13;
+
+        @media (min-width: 960px) {
+            grid-column: 9 / 13;
+        }
+        @media (min-width: 1280px) {
+            grid-column: 9 / 13;
+        }
+    }
+}
+
+.btn-secondary {
+    padding: 8px 24px;
+    border: 1px solid $accent-vermillion;
+    display: inline-block;
+    cursor: pointer;
+
+    &:hover {
+        border-color: darken($accent-vermillion, 10);
+
+        a {
+            color: darken($accent-vermillion, 10);
+            text-decoration: none;
+        }
+    }
+
+    a {
+        color: $accent-vermillion;
+    }
+}
```

Comment:
It might be a bit surprising to have such a wide casting class in this specific file. I would either include it into the `.front-matter` declaration above or move it to either owid.scss or content.scss (possibly under an `.article-content` to avoid leaking styles onto other parts of the site) - if the plan is to use that button more widely.